server/build/webpack.js

```diff
@@ -155,10 +155,33 @@ export default async function createCompiler (dir, { dev = false, quiet = false
           return { content, sourceMap }
         }
 
+        const babelRuntimePath = require.resolve('babel-runtime/package')
+          .replace(/[\\/]package\.json$/, '')
+
         const transpiled = babelCore.transform(content, {
           presets: ['es2015'],
           sourceMaps: dev ? 'both' : false,
-          inputSourceMap: sourceMap
+          inputSourceMap: sourceMap,
+          plugins: [
```

Comment:
I don't think we need them here.
Here we just need to convert import/export into commonjs. That's all.